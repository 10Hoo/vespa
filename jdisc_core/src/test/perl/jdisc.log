1359114411.180	gentleadd-lm	35172	-	org.apache.felix.framework	info	ServiceEvent REGISTERED
1359114411.192	gentleadd-lm	35172	-	jdisc_core.app-a	info	BundleEvent INSTALLED
1359114411.249	gentleadd-lm	35172	-	jdisc_core.app-a	info	BundleEvent RESOLVED
1359114411.249	gentleadd-lm	35172	-	jdisc_core.app-a	info	BundleEvent STARTED
1359114411.334	gentleadd-lm	35172	-	jdisc_core.app-a	info	BundleEvent STOPPED
1359114411.335	gentleadd-lm	35172	-	jdisc_core.app-a	info	BundleEvent UNRESOLVED
1359114411.335	gentleadd-lm	35172	-	jdisc_core.app-a	info	BundleEvent UNINSTALLED
1359114411.376	gentleadd-lm	35172	-	org.apache.felix.framework	info	ServiceEvent REGISTERED
1359114411.377	gentleadd-lm	35172	-	jdisc_core.app-a	error	my_error
1359114411.377	gentleadd-lm	35172	-	jdisc_core.app-a	warning	my_warning
1359114411.377	gentleadd-lm	35172	-	jdisc_core.app-a	info	my_info
1359114411.377	gentleadd-lm	35172	-	jdisc_core.app-a	debug	my_debug
1359114411.377	gentleadd-lm	35172	-	jdisc_core.app-a	unknown	my_unknown
1359114411.379	gentleadd-lm	35172	-	jdisc_core.app-a	info	BundleEvent INSTALLED
1359114411.383	gentleadd-lm	35172	-	jdisc_core.app-a	info	BundleEvent RESOLVED
1359114411.383	gentleadd-lm	35172	-	jdisc_core.app-a	info	BundleEvent STARTED
1359114411.389	gentleadd-lm	35172	-	jdisc_core.app-a	info	BundleEvent STOPPED
1359114411.389	gentleadd-lm	35172	-	jdisc_core.app-a	info	BundleEvent UNRESOLVED
1359114411.390	gentleadd-lm	35172	-	jdisc_core.app-a	info	BundleEvent UNINSTALLED
