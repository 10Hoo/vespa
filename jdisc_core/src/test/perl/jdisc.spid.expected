[2013-01-25 12:46:51.180] INFO    : 35172 -                org.apache.felix.framework	ServiceEvent REGISTERED
[2013-01-25 12:46:51.192] INFO    : 35172 -                jdisc_core.app-a	BundleEvent INSTALLED
[2013-01-25 12:46:51.249] INFO    : 35172 -                jdisc_core.app-a	BundleEvent RESOLVED
[2013-01-25 12:46:51.249] INFO    : 35172 -                jdisc_core.app-a	BundleEvent STARTED
[2013-01-25 12:46:51.334] INFO    : 35172 -                jdisc_core.app-a	BundleEvent STOPPED
[2013-01-25 12:46:51.335] INFO    : 35172 -                jdisc_core.app-a	BundleEvent UNRESOLVED
[2013-01-25 12:46:51.335] INFO    : 35172 -                jdisc_core.app-a	BundleEvent UNINSTALLED
[2013-01-25 12:46:51.376] INFO    : 35172 -                org.apache.felix.framework	ServiceEvent REGISTERED
[2013-01-25 12:46:51.377] ERROR   : 35172 -                jdisc_core.app-a	my_error
[2013-01-25 12:46:51.377] WARNING : 35172 -                jdisc_core.app-a	my_warning
[2013-01-25 12:46:51.377] INFO    : 35172 -                jdisc_core.app-a	my_info
[2013-01-25 12:46:51.379] INFO    : 35172 -                jdisc_core.app-a	BundleEvent INSTALLED
[2013-01-25 12:46:51.383] INFO    : 35172 -                jdisc_core.app-a	BundleEvent RESOLVED
[2013-01-25 12:46:51.383] INFO    : 35172 -                jdisc_core.app-a	BundleEvent STARTED
[2013-01-25 12:46:51.389] INFO    : 35172 -                jdisc_core.app-a	BundleEvent STOPPED
[2013-01-25 12:46:51.389] INFO    : 35172 -                jdisc_core.app-a	BundleEvent UNRESOLVED
[2013-01-25 12:46:51.390] INFO    : 35172 -                jdisc_core.app-a	BundleEvent UNINSTALLED
